.github/workflows/ruby_ci.yaml

```diff
@@ -0,0 +1,80 @@
+name: Ruby
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+env:
+  RAILS_ENV: test
+  DATABASE_URL: postgresql://postgres:@localhost/test
+  DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
+jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.5
+          bundler-cache: true
+      - name: Run rubocop
+        if: github.event_name != 'push'
+        run: bundle exec rubocop --parallel --format github
+  test_ruby:
+    runs-on: ubuntu-latest
+    needs: rubocop
+    timeout-minutes: 30
+    env:
+      BUNDLE_WITHOUT: journald:development:console:libvirt
+    services:
+      postgres:
+        image: postgres:12.1
+        ports: ['5432:5432']
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    strategy:
+      fail-fast: false
+      matrix:
+        foreman-core-branch: [develop]
+        ruby-version: [2.5]
+        node-version: [12]
```

Comment:
not sure if we need this matrix, but we can keep it,
should we also update to Ruby 2.7?